boards/imxrt1050-evkb/src/boot_header.rs

```diff
@@ -0,0 +1,410 @@
+/// Vector which contains the FCB header
```

Comment:
Can you comment where this comes from?

Reply:
Do we need the ` boards/imxrt1050-evkb/boot_header` file too?

Reply:
No, from boot_header was the vector extracted. I will delete the file now.

Reply:
I am somewhat confused by this conversation, should this file have been deleted?

Reply:
There was a binary file with a similar name that I deleted before. This is the file used now and included the main.rs of the board for the boot header.